lib/types/btrfs.nix

```diff
@@ -91,6 +91,8 @@
         let
           subvolMounts = lib.concatMapAttrs
             (_: subvol:
+            lib.warnIf (subvol.mountOptions != [] && subvol.mountpoint == null)
```

Comment:
This actually triggers the warning all the time since mountOptions.default is [ "defaults" ] and not the empty list... 
Is there a way to get the corresponding Option type default value for the comparison here?

Reply:
`options.subvolumes.[name].default` should get it, I think. But it would probably be better to just check if it's set by `options.subvolumes.[name].highestPrio < (mkOptionDefault {}).priority`. That will tell if it's set at all aside from the option's default value, even if the value is set to what happens to be the default

Reply:
Uh I think checking against the default value would be better. Since we reeval the config in the tests and then we could run in weird behavior

Reply:
What weird behavior?

I'm not sure I am able to come up with something weird that would happen (re-eval doesn't matter, this just checks what the highest option priority is set at all which only changes when someone sets it either without a priority or with a higher priority manually)

Fwiw this is how checking whether an option changed from the default is done in nixpkgs a few places

Reply:
well I pass the whole config.disko.devices into config.disko.devices again (of the test target system) so everything would be set to the default value without it still being the default value.

Reply:
Yeah but that won't change the priority value, right? (from the special type `{ _type = "override"; priority = ...; content = ...; }`)

Reply:
Oh you mean the merged config values. Huh, yeah I guess that could do something weird. Yeah I guess comparing against the default is fine then

Reply:
I found a way to access the default value using `(options.subvolumes.type.getSubOptions []).mountOptions.default`, if you think there is a simpler way, please let me know.